html-css/override-extend-variables/step3-instructions.md

```diff
@@ -1,40 +1,37 @@
-## Extend a PatternFly component to achieve a new design
-
-Now that we have practiced overriding variables, let's practice extending PatternFly's system to create a new element in the card. Let's add a separator element to the card.
```

Comment:
sb: Now that you have practiced overriding variables, practice extending PatternFly's system to create a new element in the card. You will add a separator element to the card.